components/CardsLayout.js

```diff
@@ -1,13 +1,49 @@
-import React from 'react';
+import { AiOutlineLeft, AiOutlineRight } from 'react-icons/ai';
+import React, { useRef } from 'react';
+import SVGCardBackground from './SVGCardBackground';
+import { usePosition } from 'hooks/usePosition';
 
 export default function CardsLayout({ children, description }) {
+  const ref = useRef();
+  const { hasItemsOnLeft, hasItemsOnRight, scrollRight, scrollLeft } =
+    usePosition(ref);
   return (
-    <div className="text-center hero-content md:m-auto">
-      <div className="w-lg">
-        <h2 className="pb-20 text-3xl md:text-4xl">{description}</h2>
-        <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 md:gap-x-8 gap-y-5 justify-center align-center content-center">
-          {children}
-        </div>
+    <div className="w-full py-20 relative ">
+      <h2 className="pb-20 text-3xl md:text-4xl z-10 text-center">
+        {description}
+      </h2>
+      <div className="absolute top-0 left-0 h-full w-full overflow-hidden flex justify-center items-center z-0">
```

Comment:
This looks like it works almost perfectly. Maybe we can make the size scale with an `object-fit: cover`, or some similar html attribute that will make it scale with larger screen sizes but has a `min-width` so it holds its size otherwise.

Reply:
I will try.